hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -269,6 +269,14 @@
 	private final Map subclassPropertyAliases = new HashMap();
 	private final Map subclassPropertyColumnNames = new HashMap();
 
```

Comment:
WRT "PropertyMapping SPI", I'd expect that to change dramatically in 6.0 to the point where PropertyMapping is not really needed (although it does remind me to make sure we define a "property path search" SPI over the AttributeContainer).